Apps/APN/README.md

```diff
@@ -0,0 +1,153 @@
+# Ami app
```

Comment:
This file looks like it was copied from the Ami app. I would delete it from this PR.